### DISCUSSION TITLE
why wont this compile

### DISCUSSION THREAD

**Discussion by osgsonnie** [Cyfrin/foundry-full-course-cu#1402] [General]
![Screenshot (196)](https://github.com/Cyfrin/foundry-full-course-f23/assets/127070143/b0d8f74b-7741-4417-a36f-c1e0b1db97f8)

**Comment by osgsonnie** [comment]
I have no clue why remix will not allow //SDPX-License-Identifier: MIT compile at the top

**Comment by heheboii11** [comment]
Ig you have done importing wrong, try doing 
import{PriceConverter} from "../PriceConverter.sol";
